Shared/Samples/Validate utility network topology/ValidateUtilityNetworkTopologyView.Model.swift

```diff
@@ -0,0 +1,442 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension ValidateUtilityNetworkTopologyView {
+    /// The view model for the sample.
+    @MainActor
+    class Model: ObservableObject {
+        // MARK: Properties
+        
+        /// A map with no specified style.
+        let map = Map()
+        
+        /// The graphics overlay for the starting location graphic.
+        let graphicsOverlay: GraphicsOverlay = {
+            let greenCrossSymbol = SimpleMarkerSymbol(style: .cross, color: .green, size: 25)
+            let graphic = Graphic(symbol: greenCrossSymbol)
+            return GraphicsOverlay(graphics: [graphic])
+        }()
+        
+        /// The utility network for the sample.
+        private var utilityNetwork: UtilityNetwork!
+        
+        /// The trace parameters for tracing with the utility network.
+        private var traceParameters: UtilityTraceParameters!
+        
+        /// The feature currently being edited.
+        private(set) var feature: ArcGISFeature?
+        
+        /// The feature's field currently being edited.
+        private(set) var field: Field?
+        
+        /// The coded values from the field's domain.
+        private(set) var fieldValueOptions: [CodedValue] = []
+        
+        /// The selected field coded value.
+        @Published var selectedFieldValue: CodedValue?
+        
+        /// The text representing the current status.
+        @Published var statusMessage = ""
+        
+        /// A Boolean value indicating whether the current state of the utility network can be obtained.
+        @Published private(set) var canGetState = false
+        
+        /// A Boolean value indicating whether a trace can be run.
+        @Published private(set) var canTrace = false
+        
+        /// A Boolean value indicating whether the utility network topology can be validated.
+        @Published private(set) var canValidateNetworkTopology = false
+        
+        /// A Boolean value indicating whether there is a selection that can be cleared.
+        @Published private(set) var canClearSelection = false
+        
+        deinit {
+            ArcGISEnvironment.authenticationManager.arcGISCredentialStore.removeAll()
+        }
+        
+        // MARK: Methods
+        
+        /// Gets the current state of the utility network and updates the status with the results.
+        func getState() async throws {
+            statusMessage = "Getting utility network state..."
```

Comment:
Here and other locations with "…" in this file.

```suggestion
            statusMessage = "Getting utility network state…"
```